src/tsickle.ts

```diff
@@ -1967,7 +1967,15 @@ export function emitWithTsickle(
           content = combineSourceMaps(program, fileName, content);
         }
         if (host.addDtsClutzAliases && isDtsFileName(fileName) && sourceFiles) {
-          content = addClutzAliases(fileName, content, sourceFiles, typeChecker, host);
+          // Only bundle emits pass more than one source file for .d.ts writes. Bundle emits however
+          // are not supported by tsickle, as we cannot annotate them for Closure in any meaningful
+          // way anyway.
+          if (!sourceFiles || sourceFiles.length > 1) {
```

Comment:
Are upstream d.ts files (including lib.d.ts) not source files here?

Reply:
they are not, this is just the single file that the emit happens for.